projects/schematics/src/add-spartacus/spartacus-features.ts

```diff
@@ -320,6 +320,18 @@ function configureSpartacusModules(
         });
       });
 
+      [`ExternalRoutesModule.forRoot(),`].forEach((content) => {
```

Comment:
Can we also add comment before this module to separate it from Event Modules that are added just before it?